src/app/(reskin)/components/forms/deposit/deposit-form.tsx

```diff
@@ -235,6 +236,7 @@ export const DepositForm = () => {
                 })}
               </FormButton>
             </div>
+            {isMobile && <LearnAboutSBTC />}
```

Comment:
Having the banner here seems to make it not visible in the "status" view
<img width="421" alt="image" src="https://github.com/user-attachments/assets/98118242-8e00-4033-878c-5ec8772f3c55" />

Is it fine?